package.json

```
@@ -40,7 +40,7 @@
     "prestart": "run-s generate:* ",
     "pretest": "run-s generate:* ",
     "prettier": "prettier --check \"**/*.{ts,js,mjs,json,md,yml}\"",
```

```suggestion
    "prettier": "prettier --cache --check \"**/*.{ts,js,mjs,json,md,yml}\"",
```

this will also solve caching for the `lint` command